ui/components/multichain-accounts/multichain-accounts-tree/multichain-accounts-tree.tsx

```diff
@@ -85,8 +85,11 @@ export const MultichainAccountsTree = ({
             <ButtonLink
               size={ButtonLinkSize.Sm}
               color={TextColor.primaryDefault}
-              fontWeight={FontWeight.Normal}
+              fontWeight={FontWeight.Medium}
```

Comment:
perhaps we can do something like `fontWeight={FontWeight.Medium *0.5}` or whatever number gives us the correct value. This way the props can be memoized? Are we sure that there is no `FontWeight` that gives us the desired result?

Reply:
Sorry I mis interpreted this. You are correct to use `fontWeight={FontWeight.Medium}`. I was trying to find a way to remove the 
```
style={{
                fontSize: '0.875rem',
              }}
```